eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -86,16 +86,21 @@ function ParseMavenPackage($pkg, $workingDirectory) {
   $pkgId = $contentXML.project.artifactId
   $pkgVersion = $contentXML.project.version
   $groupId = if ($contentXML.project.groupId -eq $null) { $contentXML.project.parent.groupId } else { $contentXML.project.groupId }
+  $releaseNotes = ""
+  $readmeContent = ""
 
   # if it's a snapshot. return $null (as we don't want to create tags for this, but we also don't want to fail)
   if ($pkgVersion.Contains("SNAPSHOT")) {
     return $null
   }
 
-  $releaseNotes = &"${PSScriptRoot}/../Extract-ReleaseNotes.ps1" -ChangeLogLocation @(Get-ChildItem -Path $pkg.DirectoryName -Recurse -Include "$($pkg.Basename)-changelog.md")[0]
+  $changeLogLoc = @(Get-ChildItem -Path $pkg.DirectoryName -Recurse -Include "$($pkg.Basename)-changelog.md")[0]
```

Comment:
These paths seem like a good place to use @chidozieononiwu new module that gives out the path information for our modules. However these seem to use a slightly different convention which our common modules should also be aware of.